src/actors/upgrade/common/list_content_pkg/list_content_pkg.py

```diff
@@ -43,13 +42,21 @@
                 if pkg not in pkgs:
                     pkgs.append(pkg)
             else:
-                error.append({keys['err_ctx']: context,
-                              keys['err_value']: out.rstrip()})
+                not_packaged.append(content)
 
 out = {}
 if pkgs:
     out.update({keys['out']: [{'value': pkgs}]})
-if error:
-    out.update({keys['err_out']: [{'value': error}]})
+
+if not_packaged:
+    check_result = [{
+        'check_actor': keys['name'],
+        'check_action': context,
+        'status': 'FAIL',
+        'summary': 'Path is not owned by any package',
+        'params': not_packaged
+    }]
+    check_out = [{'checks': check_result}]
+    out.update({keys['check_out']: check_out})
 
 print(json.dumps(out))
```

Comment:
Just reminder for others that used schema is not final solution and in future it will be different. We can expect later something like:
```diff
...
+        'status': 'inspection'
+        'summary': 'Detected <lang> modules that are not owned by any package.',
+        'body': You have installed <lang> modules that are not owned by any package. These will not be handled during upgrade. List of the untracked modules: {{ html_list .params }}
...
```
(that's really rough example)